### ISSUE TITLE
your backend official link is not working (october 2024)

### ISSUE THREAD

**Issue by chritomix** [reflex-dev/reflex-web#1026]
Hi, just starting. but it seems your website is not working for the last few days. each time i look or wish to test sth on your website, i have that red message:
Cannot connect to server: websocket error. Check if server is reachable at wss://rxweb-prod-reflex-web.fly.dev/_event

can you fix it? thank you!

**Comment by Alek99** [comment]
What country are you in? I haven't been seen this in the Bay Area

**Comment by chritomix** [comment]
France
![image](https://github.com/user-attachments/assets/58c0c2cf-e303-4696-8f0a-d2627ba75f9d)

and the examples like: an example make this count (https://reflex.dev/docs/getting-started/introduction/) is not working. 
i am using firefox, on windows

**Comment by GOvEy1nw** [comment]
Yep I'm getting the same errors, making browsing the docs very tricky: Cannot connect to server: timeout.
Check if server is reachable at wss://rxweb-prod-reflex-web.fly.dev/_event

**Comment by benedikt-bartscher** [comment]
Same issue from Germany, vpn to US works

**Comment by benedikt-bartscher** [comment]
Just noticed: Sometimes it works from Germany as well, but very slow and flaky (unusable). US is much faster and stable
